src/view/pages/Farmer/tabs/Info/index.tsx

```diff
@@ -66,22 +51,12 @@ class FarmerInformation extends React.Component<OwnProps, OwnState> {
     this.setState(() => ({ mode: newMode }));
   }
 
-  private updatePaymentCycle = (value: string) => {
-    // this.setState(() => ({ selectedPaymentCycle: value }));
-  }
-
-  private updatePaymentMethod = (value: string) => {
-    // this.setState(() => ({ selectedPaymentMethod: value }));
-  }
-
   /**
    * Render method for Farmer Information
    */
   public render() {
     if (this.state.mode === modes.view) {
       return <ViewFarmer {...this.initialProps} modeHandler={this.changeMode} />;
-    } else if (this.state.mode === modes.add) {
-      return <AddFarmer {...this.initialProps} modeHandler={this.changeMode} updatePaymentCycle={this.updatePaymentCycle} updatePaymentMethod={this.updatePaymentMethod} />;
     } else if (this.state.mode === modes.edit) {
       return <EditFarmer {...this.initialProps} modeHandler={this.changeMode} />;
     }
```

Comment:
Please include a default case that throws an exception, just in case someone does break types we don't want to render an empty page with no error